Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue7176.cs

```diff
@@ -0,0 +1,78 @@
+﻿using Xamarin.Forms.CustomAttributes;
+using Xamarin.Forms.Internals;
+
+#if UITEST
+using NUnit.Framework;
+using Xamarin.UITest;
+using Xamarin.Forms.Core.UITests;
+using Xamarin.UITest.Queries;
+#endif
+
+namespace Xamarin.Forms.Controls.Issues
+{
+	[Preserve(AllMembers = true)]
+	[Issue(IssueTracker.Github, 7176, "[Enhancement] Label Autofit Font", PlatformAffected.Android | PlatformAffected.iOS)]
+#if UITEST
+	[Category(UITestCategories.Label)]
+#endif
+	public class Issue7176 : TestContentPage
+	{
+		const string GetFontSize = "getTextSize";
+		const string GetAutoSizeMode = "getAutoSizeTextType";
+		const string AutoFitLabel = "Autofit Label";
+		const string AutoFitModeButton = "Autofit Mode Button";
+
+		protected override void Init()
+		{
+			var autofitLabel = new Label
+			{
+				AutomationId = AutoFitLabel,
+				AutoFit = true,
+				Text = "Welcome to xamarin forms!!",
+				VerticalOptions = LayoutOptions.FillAndExpand,
+				HorizontalOptions = LayoutOptions.FillAndExpand,
+				BackgroundColor = Color.LightBlue
+			};
+
+			Content = new StackLayout()
+			{
+				Children =
+				{
+					new ApiLabel(),
+					autofitLabel,
+					new Button
+					{
+						Text = "Change autofit mode",
+						AutomationId = AutoFitModeButton,
+						Command = new Command(() => autofitLabel.AutoFit = !autofitLabel.AutoFit)
+					}
+				}
+			};
+		}
+
+#if UITEST && __ANDROID__
```

Comment:
Needs iOS test, can we maybe do without the Android specific invokes and make the test cross-platform?

Reply:
we can write IOS specifics invokes too, but i dont have a MAC.. so i can do by documentation, without testing properly 😔

Reply:
I noticed you rented a Mac for this, awesome! :D Thank you so much for your dedication on this, we really appreciate it!

In that light, would you be able to make the test work for iOS too? :)